test/e2e/accounts/common.ts

```diff
@@ -372,3 +372,15 @@ export async function signData(
     });
   }
 }
+
+export async function createBtcAccount(driver: Driver) {
+  await driver.clickElement('[data-testid="account-menu-icon"]');
+  await driver.clickElement(
+    '[data-testid="multichain-account-menu-popover-action-button"]',
+  );
+  await driver.clickElement({
+    text: 'Add a new Bitcoin account (Beta)',
```

Comment:
Similarly to other part of this PR, we could use this (you will need to import `messages.json` too, so this suggestion is incomplete):

```suggestion
    text: messages.addNewBitcoinAccount.message,
```